### PR TITLE
Cross link documentation for creating dedicated nodes with special hardware and ExtendedResourceToleration admission controller.

### PR DESCRIPTION
**Pull request by mindprince**
Fixes kubernetes/kubernetes#62138

### CODE REVIEW

File: docs/concepts/configuration/taint-and-toleration.md [hunk 1]
```diff
@@ -171,15 +171,18 @@ hardware (e.g. `kubectl taint nodes nodename special=true:NoSchedule` or
...
-For example, the admission controller could use
-some characteristic(s) of the pod to determine that the pod should be allowed to use
-the special nodes and hence the admission controller should add the toleration.
-To ensure that the pods that need
-the special hardware *only* schedule onto the nodes that have the special hardware, you will need some
-additional mechanism, e.g. you could represent the special resource using
-[extended resources](/docs/concepts/configuration/manage-compute-resources-container/#extended-resources)
-and request it as a resource in the PodSpec, or you could label the nodes that have
-the special hardware and use node affinity on the pods that need the hardware.
+For example, you can use [Extended
```
Comment (vishh) [inline docs/concepts/configuration/taint-and-toleration.md]: Mention that we *recommend* using the extended admission controller to guarantee portability for extended resources.

Comment (heckj) [reply docs/concepts/configuration/taint-and-toleration.md]: @vishh I added a comment to this effect as a Note callout - Our style guide is set up to discourage the use of "we" (https://kubernetes.io/docs/home/contribute/style-guide/#avoid-using-we)

File: docs/concepts/configuration/taint-and-toleration.md [hunk 2]
```diff
@@ -171,15 +171,22 @@ hardware (e.g. `kubectl taint nodes nodename special=true:NoSchedule` or
...
+[ExtendedResourceToleration](/docs/admin/admission-controllers/#extendedresourcetoleration)
+admission controller. Now, because the nodes are tainted, no pods without the
+toleration will schedule on them. But when you submit a pod that requests the
+extended resource, the `ExtendedResourceToleration` admission controller will
+automatically add the correct toleration to the pod and that pod will schedule
+on the special hardware nodes. This will make sure that these special hardware
+nodes are dedicated for pods requesting such hardware and you don't have to
+manually add tolerations to your pods.
+
+**NOTE:** You can guarantee portability for extended resources by using the extended admission controller.
```
Comment (heckj) [inline docs/concepts/configuration/taint-and-toleration.md]: @vishh does this cover what you're after?

File: docs/concepts/configuration/taint-and-toleration.md (hunk 1 above)
Comment (mindprince) [reply docs/concepts/configuration/taint-and-toleration.md]: Done. Added `it is recommended...`.

File: docs/concepts/configuration/taint-and-toleration.md (hunk 2 above)
Comment (vishh) [reply docs/concepts/configuration/taint-and-toleration.md]: @heckj nope. I want a more definitive stance on using the admission controller mentioned in this PR.

**Pull request merged**